src/server.go

```diff
@@ -59,35 +93,12 @@ type jobServer struct {
 //  4. Creates a client of the Dispatcher service.
 //  5. Constructs a data message to send back to the dispatcher.
 //  6. Sends the data message using the "Send" method of the Dispatcher service.
-func (s *jobServer) Send(ctx context.Context, d *pb.Data) (*pb.Receipt, error) {
-	go func() {
-		log.Infoln("Processing received yaml data")
-		commandOutput := processSignedScript(d.GetContent())
-
-		// Dial the Dispatcher and call "Finish"
-		conn, err := grpc.Dial(
-			yggdDispatchSocketAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
-		if err != nil {
-			log.Error(err)
-		}
-		defer conn.Close()
+func (s *jobServer) Send(_ context.Context, d *pb.Data) (*pb.Receipt, error) {
```

Comment:
Since we are not using context anymore, do we still need it here?

Reply:
no we don't, even before we were just reassigning it with new context but golang will complain about wrong method if I remove it :) 

![image](https://github.com/oamg/rhc-worker-bash/assets/19702477/569a5ffe-b435-4cb6-9911-5ac7dc6b8285)

Reply:
Aha, I see. You're right! 

Thanks for the clarification.